src/app/components/cohort-selection/cohort-selection.tpl.html

```diff
@@ -9,7 +9,7 @@
                 popupDelay="1000"><span class="glyphicon glyphicon-cloud-upload"></span></button>
         <button type="button"
                 class="btn btn-secondary"
-                uib-tooltip="Load cohorts"
+                uib-tooltip="Load cohorts (yet to be implemented)"
```

Comment:
As discussed, we will just simply disable this button until the feature is working